drivers/flash/flash_stm32_xspi.c

```diff
@@ -0,0 +1,2076 @@
+/*
+ * Copyright (c) 2022-2024 STMicroelectronics
+ * Copyright (c) 2022 Georgij Cernysiov
```

Comment:
Is this Copyright correct?

Reply:
new version with only  "2024 STMicroelectronics"